ts/window.d.ts

```diff
@@ -98,5 +100,10 @@ declare global {
     darkTheme: DefaultTheme;
     LokiPushNotificationServer: any;
     LokiPushNotificationServerApi: any;
+
+    getConversationController: () => ConversationController;
+    getOrCreateAndWait: () => Promise<ConversationModel>;
+
+    commitProfileEdits: (string, string) => void;
```

Comment:
those must be removed too once you moved those function to some typescript function you can import. Ask me if needed